pkg/proxy/replica.go

```diff
@@ -30,7 +31,30 @@ func (p *Proxy) ReplicaList(ctx context.Context, req *rpc.ProxyEngineRequest) (r
 		ReplicaList: &eptypes.ReplicaListReply{},
 	}
 
-	// TODO
+	c, err := eclient.NewControllerClient(req.Address)
+	if err != nil {
+		return resp, err
```

Comment:
NIT: Explicitly returning `nil` rather than `resp` if there is an error. Similar for other implementations

Reply:
I don't think gRPC server can return a nil message.

Reply:
Updated.